File: engine/engine-model/src/main/scala/tools/packet/maker/MakerCrystalResult.scala
package tools.packet.maker

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class MakerCrystalResult(private var _itemIdGained: Int, private var _itemIdLost: Int) extends PacketInput {
  def itemIdGained: Int = _itemIdGained

  def itemIdLost: Int = _itemIdLost

  override def opcode(): SendOpcode = SendOpcode.MAKER_RESULT
}

File: engine/engine-model/src/main/java/tools/packet/maker/MakerCrystalResult.java
package tools.packet.maker;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record MakerCrystalResult(Integer itemIdGained, Integer itemIdLost) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.MAKER_RESULT;
   }
}